docs/migrations/v2-to-v3.md

```diff
@@ -34,16 +34,23 @@ If the chain will adopt ICS27, it must set the appropriate params during the exe
 ```go
 app.UpgradeKeeper.SetUpgradeHandler("v3",
     func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
-        // set ICS27 Host submodule params
-        app.ICAHostKeeper.SetParams(ctx, icahosttypes.Params{
-            HostEnabled: true, 
-            AllowMessages: []string{"/cosmos.bank.v1beta1.MsgSend", ...], 
-        })
+        // set the ICS27 consensus version so InitGenesis is not run
+        fromVM[icatypes.ModuleName] = icamodule.ConsensusVersion()
```

Comment:
where is `fromVM` defined? Shouldn't it be `vm`?

Reply:
yea sorry, in SDK docs they usually call it fromVM. I'll update vm to be fromVM